athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/domain/Split.java

```diff
@@ -239,6 +239,17 @@ public Builder withEncryptionKey(EncryptionKey key)
             return this;
         }
 
+        /**
+         * Apply a map of properties to the split
+         * @param properties
+         * @return
+         */
+        public Builder applyProperties(Map<String, String> properties)
+        {
+            properties.entrySet().forEach(entry -> add(entry.getKey(), entry.getValue()));
```

Comment:
can you rename properties from input to inputProperties or something. Right now the name is the same.

After that can we just this.properties.addAll(properties)

Reply:
The SDK should be the only user of this interface; I don't think that would be a problem.